CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -8,7 +8,8 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 
 ### Package `clientv3`
 - Add [client backoff and retry config options](https://github.com/etcd-io/etcd/pull/17363).
-
+### Package `client`
+- Add [OpenBSD support](https://github.com/etcd-io/etcd/pull/15812)
```

Comment:
We are not officially supporting OpenBSD yet.

```suggestion
- [Ignore SetKeepAlivePeriod errors on OpenBSD](https://github.com/etcd-io/etcd/pull/15812)
```

Reply:
done 👍🏽